### ISSUE TITLE
Backup to USB drive instead internal drive

### ISSUE THREAD

**Issue by Monacoslo** [sabeechen/hassio-google-drive-backup#537]
I run HA on mini PC with only 16gb ssd. How can I force addon to make snapshots on HDD attached to SSD?

**Comment by sabeechen** [comment]
TL;DR: To the best of my knowledge, there is no way to do this.

Home Assistant's supervisor is the one who actually creates the backups, and I'm not aware of a way to change the place where it creates backups.  The best I can recommend is to "minimize" the amount of space the backups use locally by checking  "Delete Backups after Upload" in the addon settings.

This way it will only ever create one backup locally and then delete it after being uploaded to Google Drive (ie before the next snapshot is created). Naturally this will only work if you have enough free space to store at least one snapshot.  If you don't have that you'll need to make your snapshots smaller to get them to fit.  You may also want to consider [requesting this feature in the supervisor's project](https://github.com/home-assistant/supervisor), but given what I know of how the supervisor is implemented I think its unlikely the maintainers would want to do it.
